documentation/modules/ref_concurrent-migrations-faster-migrations.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migrating_your_virtual_machines/assemblies/assembly_mtv-performance-recommendations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref_concurrent-migrations-faster-migrations_{context}"]
+= Migrations are completed faster when migrating multiple VMs concurrently
+
+[role="_abstract"]
+When multiple VMs from a specific ESXi host are to be migrated, starting concurrent migrations for multiple VMs leads to faster migration times. 
```

Comment:
Suggestion "to be migrated by using {project-first}," -- as it is, this module doesn't mention MTV.